app/multitenant/dynamo_collector.go

```diff
@@ -59,6 +54,12 @@ var (
 		Help:      "Size of data read / written from dynamodb.",
 	}, []string{"method"})
 
+	inProcessCacheCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "scope",
+		Name:      "dynamo_cache", // TODO(jml): Rename this monitoring variable.
```

Comment:
As its renamed from dynamo_cache(_hits/_miss) already, might as well go the whole hog.